tests/api.ts

```diff
@@ -457,31 +456,25 @@ export async function collectProfitOfCredixLpDepository(
     uxdClient.createCollectProfitOfCredixLpDepositoryInstruction(
       controller,
       depository,
-      authority.publicKey,
-      TXN_OPTS,
-      payer.publicKey
+      payer.publicKey,
+      TXN_OPTS
     );
   let signers = [];
   let tx = new Transaction();
-
-  const [authorityCollateralAta] = findATAAddrSync(
-    authority.publicKey,
+  const [payerCollateralAta] = findATAAddrSync(
+    payer.publicKey,
```

Comment:
the Ata should be the created for the profitBeneficiary instead?

Reply:
In this case, since we need to be the owner of the account to create the account im using the payer as the beneficiary

Reply:
I will clarify this test tho by passing the profitBeneficiary from somewhere else instead 👌 this should be more clear